### DISCUSSION TITLE
Changes to ppooll-quickstart.pdf

### DISCUSSION THREAD

**Discussion by gregdl** [ppooll-dev/ppooll#112] [User Feedback]
The ppooll quickstart [pdf](https://ppooll.klingt.org/wp-content/uploads/2023/07/ppooll-quickstart.pdf) on page 20 suggests that users should "chat with developers", "seek technical assistance" and "report bugs" on a Discord server.

Is this the best advice to give people using ppooll, especially those using it for the first time?
- are the developers of ppooll actively participating in this Discord and actively responding to users seeking technical assistance and reporting potential bugs?
- is the Discord the correct place for people to report bugs? I would have assumed GitHub would be a better channel for this.

> since 2022, we’ve hosted a Discord server, where you can chat with developers, seek technical assistance, report bugs, share acts, contribute, discuss and explore ppooll together

**Comment by chausch** [comment]
> The ppooll quickstart [pdf](https://ppooll.klingt.org/wp-content/uploads/2023/07/ppooll-quickstart.pdf) on page 20 suggests that users should "chat with developers", "seek technical assistance" and "report bugs" on a Discord server.
> 
> Is this the best advice to give people using ppooll, especially those using it for the first time?
> 
>     * are the developers of ppooll actively participating in this Discord and actively responding to users seeking technical assistance and reporting potential bugs?
> 
>     * is the Discord the correct place for people to report bugs? I would have assumed GitHub would be a better channel for this.
> 
> 
> > since 2022, we’ve hosted a Discord server, where you can chat with developers, seek technical assistance, report bugs, share acts, contribute, discuss and explore ppooll together

thanks for the input. yes, github is the place for bug reports! seein the guide for the first time now – was it written by you, @elinsound ?

**Reply by elinsound** [reply]
yes, i wrote this pdf to answer a lot of quick and common questions that users had, and also to provide a written alternative to the brief video tutorial that i put together for youtube. 

i wrote it rather quickly, sorry. there are several mentions of the github in there, but i can amend it, sure. 

in the meantime, it may be wise to acknowledge that the discord server is where the ppooll community have chosen to network, discuss and help one another. i have sent several mass messages to the userbase in order to direct them to report issues here. you can lead a horse to water, but...

**Comment by gregdl** [comment]
thanks @chausch & @elinsound 

the discord server has become an excellent community of people engaged, excited, and inspired by using ppooll. 

there are many experienced users offering their time, serving as a support system to new users who encounter friction and/or bugs with using ppooll, but perhaps there needs to be more reinforcement of pointing people here for things that the core developers should attend to.

maybe a suggestion besides editing the pdf would be to add a note to the ppooll website itself along with the info act available from host asking users to report bugs and suggestions for improvements to the github along w/ a link pointing to gh. 

there could be some acknowledgement that the discord community is welcoming for all ppooll users but that it is not related to ppooll development (it's "unofficial" maybe, not sure of correct vocabulary).
